frontend/js/components/CardTeam.js

```diff
@@ -2,11 +2,11 @@ import React from 'react';
 
 import CardPokemon from 'components/CardPokemon';
 
-function CardTeam({ pokemons }) {
+function CardTeam(props) {
```

Comment:
Based on my previous comments this will need to be updated

Reply:
Thanks!